### DISCUSSION TITLE
where is the AGH log?

### DISCUSSION THREAD

**Discussion by Stooovie** [hassio-addons/addon-adguard-home#277] [Q&A] [answered]
I have an AGH issue and the dev asks me for AGH log. I set the addon log_level to debug but the log from Supervisor - Addon - AGH - Log just cuts short for whatever reason.

Where is the location of the physical log and is there another way of getting it other than from HA UI?

Thanks.

**Comment by frenck** [comment, marked as answer]
> Where is the location of the physical log and is there another way of getting it other than from HA UI?

There isn't one, log information is redirected to the Docker output and handled by the Supervisor, which one can view in the add-on log.
